src/MICore/CommandFactories/gdb.cs

```diff
@@ -258,5 +263,17 @@ private static string TypeBySize(uint size)
             }
             return TargetArchitecture.Unknown;
         }
+
+        public override async Task Signal(string sig)
```

Comment:
What other code paths? This is simply executing the debugger's 'signal' command via MI. I think any other code paths must be executing a shell command so shouldn't affect the MICommandFactory.